### PR TITLE
feat: created the footer

### LINKED ISSUE #43: [feat] Create footer of the website
**Is your feature request related to a problem? Please describe.**
create footer with one logo and text provided in the screen shot.
footer

**Describe the solution you'd like**
create exactly similar of what we provided in the screen shot.

**Additional context**
the logo of the footer is this you can download it attach in your code

go-hub-logo

### PR DESCRIPTION
**Pull request by ductrl**
## What does this PR do?

Fixes #43 

## Type of change

- New feature (non-breaking change which adds functionality)

## Mandatory Tasks

- [X] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I have read the [contributing guide](https://github.com/DhanrajKamble/ThePokedex/blob/master/CONTRIBUTING.MD)
- My code follow the style guidelines of this project
- I commented my code, particularly in hard-to-understand areas
- I haved checked if my PR needs changes to the documentation
- I have checked if my changes generate no new warnings

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,299 +1,328 @@
...
+  height: 100vh;
+  width: 100vw;
+  overflow-x: hidden;
+  overflow-y: auto;
+  position: relative;
 }
-body{
-    background-color:#F0F0E5;
+body {
+  background-color: #f0f0e5;
```
Comment (Rahufkhan) [inline style.css:17]: don't change the code #F0F0E5 to #f0f0e5 both one and some leave as it is

File: style.css [hunk 2]
```diff
@@ -1,299 +1,328 @@
...
+  text-align: center;
 }
-.category-name a:nth-child(2){
-    color: rgb(63, 62, 62);
-    font-size: 0.9rem;
-    font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif ;
+
+#footer-logo {
+  /* Setting the size for the logo */
+  width: 10vw;
```
Comment (Rahufkhan) [inline style.css]: fix logo width in responsive mobile screen size. logo is decreasing its width wen we shift to mobile size

**Comment by Rahufkhan** [comment]
don't change the previous code of css just add the footer css and footer responsive css to it.

**Comment by Rahufkhan** [comment]
increase logo size to mobile screen size

**Comment by Rahufkhan** [comment]
in mobile size screen footer font size should be like this.

mobile_size_footer

File: index.html [hunk 3]
```diff
@@ -288,7 +288,12 @@ <h1>Fun and Tools</h1><hr>
 ============================================= ------>
 <div class="footer-container">
     <footer>
-        <p>All ©️copyrights reserved ThePokedex 2023...</p> 
+        <!-- Linking the site to the logo -->
+        <a href="https://the-pokedex-nu.vercel.app/">
+            <img src="https://user-images.githubusercontent.com/62764275/282056861-444edccd-a670-421a-a1f0-45caed15582d.svg" alt="GO Hub Logo" id="footer-logo">
```
Comment (Rahufkhan) [inline index.html]: you can upload footer logo image file and upload it in the images folder by downloading it in the issue #43

**Comment by ductrl** [comment]
I updated the files based on your feedbacks. I'm pretty unexperienced in coding so I'd love to hear more feedbacks from you!

**Comment by Rahufkhan** [comment]
No worries, bro, that's what you learn from your mistake. Just keep it up. You have done great work; I will merge it. After all it is a beginner-friendly project.

**Comment by ductrl** [comment]
Thank you so much for the kind words. Please let me know if I can help with anything in the future. Wish you the best of luck with the project!

**Pull request merged**